nexus/test-utils/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 //! Integration testing facilities for Nexus
 
+#[cfg(feature = "omicron-dev")]
```

Comment:
I threw this in because I saw a spurious warning about this import when doing `cargo nextest run -p omicron-omdb`.